index.html

```diff
@@ -80,17 +91,18 @@ <h2>Join thousands of others...</h2>
 			</div>
 		</section>
 
-		<!-- SECTION 2 -->
-		<section class="flex-section-container-one-item">
-			<div class="carousel-container margin-bottom-5">
-				<h2>
-					Check out our beautiful 🌵 plants...
-				</h2>
-				<div class="image-container">
-					<img id="left-arrow" src="/assets/left-arrow.png" />
-					<img src="assets/carousel-photos/plants-photo1.jpeg" alt="Photo of the flower" />
-					<img id="right-arrow" src="/assets/right-arrow.png" />
-				</div>
+		<!-- SECTION 2 CAROUSEL -->
+		<section class="carousel-container">
+			<h2>Check out our 🌵 plants</h2>
+			<div class="carousel">
+				<div class="left-button"><</div>
+				<img src="assets/carousel-photos/plant-photo1.jpeg" alt="Photo of the flower" />
+				<img src="assets/carousel-photos/plant-photo2.jpeg" alt="Photo of the flower" />
+				<img src="assets/carousel-photos/plant-photo3.jpeg" alt="Photo of the flower" />
```

Comment:
Plant-photo3 is a bit shorter than the other images and changes the location of the buttons on the screen